### ISSUE TITLE
ISW - Wizard Step: Input Diagnostics

### ISSUE THREAD

**Issue by grotlue** [Graylog2/graylog2-server#20683]
Issue for building the diagnostics page.

-  Shows a status of the input with
   - Input name
   - Input description (stretch goal)
   - node it is running on
   - input specific details (only for listener inputs)
     - bind address (if available)
     - port (if available)
     - show info copy to check if UCP or TCP
- Shows a button "Show Received Messages"
  - clicking on it will forward you to the search page
  - already exists: ![Image](https://github.com/user-attachments/assets/1b0adf0d-98c7-4079-95c2-1be514b450f2)

#### Metrics:
- if any metric does not exist we show 0
- we will use the `fifteen_minute` property of the metrics `rate` object
- the metrics are automatically refetched every 5 seconds
- we will show an UI element that can manually trigger a refresh of the metrics
- Throughput (already exists): ![Image](https://github.com/user-attachments/assets/11933620-ec76-4d7d-badf-1aba1bb01923)
- Message error at input
- Message failed to process
- Message failed to index
 - Input state and detailed message(already exists in input state)
   - We show a summary of existing states of all nodes of this input (e.g.: FAILED: 2/10, RUNNING: 8/10)
     - When clicking on each state, the list expands, showing the related node ids
     - if a detailed message exists, the node id can be clicked to show it
 - Messages received by input count 
   - Descriptional text of that this metric represents 
 - Streams count and stream names ( count query for gl2_source_input and stream)

## Mock-Up

![Image](https://github.com/user-attachments/assets/0137fa0c-5fd4-4ce1-b724-22635e560ff4)

**Comment by grotlue** [comment]
To show the input name and a type specific description we will need an additional call returning input name and description based on a given type.

**Comment by grotlue** [comment]
@AntonEbel Do you know if we have the information TCP or UCP for listener inputs? bind address and port I can find in the form.

**Comment by AntonEbel** [comment]
@grotlue No, we do not provide this information and do not persist it. I don't know if we want to display this explicitly because the TCP/UDP protocol is already included in the Input Type name.

**Comment by patrickmann** [comment]
Agree with @Anton that there is not a lot of value in adding the TCP/UDP detail.

We can use existing metrics APIs for all of the numbers. So what is left that backend needs to expose? This part seems to have some new information:

> We show a summary of existing states of all nodes of this input (e.g.: FAILED: 2/10, RUNNING: 8/10)
> When clicking on each state, the list expands, showing the related node ids
> if a detailed message exists, the node id can be clicked to show it
> Messages received by input count ( Opensearch count query for gl2_source_input)
> Streams count and stream names ( count query for gl2_source_input and stream)

**Comment by grotlue** [comment]
> [@grotlue](https://github.com/grotlue) No, we do not provide this information and do not persist it. I don't know if we want to display this explicitly because the TCP/UDP protocol is already included in the Input Type name.

> Agree with [@anton](https://github.com/anton) that there is not a lot of value in adding the TCP/UDP detail.


@tellistone What is your take on the TCP/UDP detail?

**Comment by tellistone** [comment]
could be replaced with a line of ui copy reminding the user to check they are using the right one re: TCP/UDP 

Customers trip over this a surprising amount (perhaps because the app gives no feedback when this is happening)
